src/runtime/plugins/supabase.client.ts

```diff
@@ -26,6 +31,7 @@ export default defineNuxtPlugin({
 
     // Updates the session and user states through auth events
     client.auth.onAuthStateChange((_, session: Session | null) => {
+      client.realtime.setAuth(session?.access_token ?? null)
```

Comment:
What is this line for? I can't find any documentation about this `realtime.auth` method.

Reply:
Sorry, this slipped in there because there is a realtime bug that I am debugging with the Supabase team where the auth token does not correctly propagate to realtime. Will take it out for now